_translations/en.yml

```diff
@@ -285,23 +285,23 @@ en:
     walletarmory: "Armory is an advanced Bitcoin client that expands its features for Bitcoin power users. It offers many backup and encryption features, and it allows secure cold-storage on offline computers."
     walletelectrum: "Electrum's focus is speed and simplicity, with low resource usage. It uses remote servers that handle the most complicated parts of the Bitcoin system, and it allows you to recover your wallet from a secret phrase."
     walletmsigna: "mSIGNA is an advanced yet easy-to-use wallet featuring speed and simplicity, enterprise-level scalability, and strong security. It supports BIP32, multisignature transactions, offline storage, multidevice synchronization, and encrypted electronic and paper backups."
-    walletbitcoinwallet: "Bitcoin Wallet is easy to use and reliable, while also being secure and fast. Its vision is de-centralization and zero trust; no central service is needed for Bitcoin-related operations. The app is a good choice for non-technical people."
+    walletbitcoinwallet: "Bitcoin Wallet is easy to use and reliable, while also being secure and fast. Its vision is decentralization and zero trust; no central service is needed for Bitcoin-related operations. The app is a good choice for non-technical people."
     walletedgewallet: "Edge is a Segwit compatible, mobile Bitcoin Wallet making high levels of privacy, security, and decentralization very familiar and usable to the masses. Edge wallets are always automatically encrypted, backed up, and even function when Edge servers go down."
-    walletbrd: "BRD is one of the easiest wallets for new Bitcoin users. It's also extremely secure. BRD connects directly to the Bitcoin network, and uses your device's built-in hardware encryption to keep your money safe."
-    walletmyceliumwallet: "Mycelium Bitcoin Wallet for Android is designed for security, speed, and ease of use. It has unique features to manage your keys and for cold storage and offers compatibility with Trezor and others."
-    walletcoin: "Coin is a free online bitcoin wallet, which you can use to make worldwide payments for free. It makes paying with bitcoins easy and secure available anywhere on your phone or desktop."
+    walletbrd: "BRD is designed to be easy to use for new Bitcoin users. It's also secure. BRD connects directly to the Bitcoin network, and uses your device's built-in hardware encryption to keep your money safe."
```

Comment:
I suggest leaving out "designed to be".  It is in fact easy to use.

Reply:
501cf2c0